lib/pages/main/components/navigation_drawer_widget.dart

```diff
@@ -0,0 +1,217 @@
+                        style: TextStyle(fontSize: 22),
+                      ),
+                    ),
+                    _uuidWithClipboard(),
+                  ],
+                )
+              ],
+            )
+          ),
+          ListTile(
+            title: Text(MyLocalizations.of(context, 'home_tab')),
+            leading: Icon(Icons.home),
+            onTap: () {
+
+            },
+          ),
+          ListTile(
+            title: const Text('My reports'),
+            leading: Icon(Icons.file_copy),
+            onTap: () {
+              Navigator.push(
+                context,
+                MaterialPageRoute(
+                  builder: (context) => MyReportsPage()),
+              );
+            },
+          ),
+          ListTile(
+            title: Text(MyLocalizations.of(context, 'public_map_tab')),
+            leading: Icon(Icons.map),
+            onTap: () {
+              // TODO: Public map
+            },
+          ),
+          ListTile(
+            title: Text(MyLocalizations.of(context, 'guide_tab')),
+            leading: Icon(Icons.science),
+            onTap: () {
+              Navigator.push(
+                context,
+                MaterialPageRoute(
+                  builder: (context) => CampaignTutorialPage()),
+              );
+            },
+          ),
+          ListTile(
+            title: Text(MyLocalizations.of(context, 'settings_title')),
+            leading: Icon(Icons.settings),
+            onTap: () {
+              Navigator.push(
+                context,
+                MaterialPageRoute(
+                  builder: (context) => SettingsPage()),
+              );
+            },
+          ),
+          ListTile(
+            title: Text(MyLocalizations.of(context, 'info_tab')),
+            leading: Icon(Icons.info),
+            onTap: () {
+              // TODO: About page
+            },
+          ),
+          SizedBox(
+            height: 100,
+          ),
+          Center(
+            child: Column(
+              mainAxisSize: MainAxisSize.max,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                Style.bodySmall(
+                  packageInfo != null
+                      ? 'version ${packageInfo.version} (build ${packageInfo.buildNumber})'
+                      : '',
+                  fontSize: 8,
+                  textAlign: TextAlign.center),
+              ],
+            ),
+          ),
+        ],
+      ));
+  }
+
+    Widget _uuidWithClipboard(){
+    return FutureBuilder(
+      future: UserManager.getUUID(),
+      builder: (BuildContext context, AsyncSnapshot<String?> snapshot) {
+        if (snapshot.connectionState == ConnectionState.waiting) {
+          return CircularProgressIndicator();
+        } else if (snapshot.hasError) {
+          return Text('Error: ${snapshot.error}');
+        }
+
+        print(snapshot.data);
+        return Row(
+          children: [
```

Comment:
Could you add the string 'ID: ' before pasting the user uuid? (sorry, this was not in the screenshots I share with you)

Example: 'ID: 7a5c1cf0-7fe6-4a8b-81ba-67e72475742b'

'ID: ' -> color black
'7a5c1cf0-7fe6-4a8b-81ba-67e72475742b' -> color grey (whatever if it looks lighter color than 'ID')